biogeophys/FatesPlantRespPhotosynthMod.F90

```diff
@@ -25,11 +25,25 @@ module FATESPlantRespPhotosynthMod
    use FatesConstantsMod, only : r8 => fates_r8
    use FatesConstantsMod, only : itrue
    use FatesInterfaceMod, only : hlm_use_planthydro
+   use FatesInterfaceMod, only : hlm_parteh_model
    use FatesInterfaceMod, only : numpft
    use EDTypesMod, only        : maxpft
    use EDTypesMod, only        : nlevleaf
    use EDTypesMod, only        : nclmax
-   
+
+   use PRTGenericMod,          only : prt_carbon_allom_hyp
+   use PRTGenericMod,          only : carbon12_species
+   use PRTGenericMod,          only : nitrogen_species
+   use PRTGenericMod,          only : phosphorous_species
+   use PRTGenericMod,          only : leaf_organ
+   use PRTGenericMod,          only : fnrt_organ
+   use PRTGenericMod,          only : sapw_organ
+   use PRTGenericMod,          only : store_organ
+   use PRTGenericMod,          only : repro_organ
+   use PRTGenericMod,          only : struct_organ
+   use PRTGenericMod,          only : carbon12_species
+   use PRTGenericMod,          only : SetState
```

Comment:
why does one need to access 'setstate' but not 'getstate'?

Reply:
GetState is bound to the class object cohort%prt itself.  Whereas SetState is just a subroutine.  Right now there is no reason that SetState has to be its own subroutine.  I think the reason I left it that way was because there is this thing you can do where you can make a subroutine have extended versions of themselves, each of which are called in situations with different arguments.  This functionality was attactive for when the time came when we would have some hypotheses defining multiple discrete parts to each organ (e.g. leaf layers), versus hypthotheses defining just one discrete part per organ (what we have now for allocation).